### DISCUSSION TITLE
Selkie price is 7,292 mk.

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#13832] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

A shuttle tier selkie costs more than some of the tier 1 submarines.

### Reproduction steps

1. Host a server
2. Select selkie, selkie price - 7,292 mk.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/ded6a3de26d3718da8154e24a87732a6e0aba75e (commit in our private repo).
